packages/node/src/plugins/NodeFederationPlugin.ts

```diff
@@ -123,8 +121,12 @@ export const generateRemoteTemplate = (url: string, global: any) => `new Promise
         global.__remote_scope__[${JSON.stringify(global)}].__initialized = true
       }
     }
-    if (!global.__remote_scope__[${JSON.stringify(global)}].__initialized) {
-      proxy.init(__webpack_require__.S.default)
+    try {
```

Comment:
Might have gone overboard with try catch on init api. 

Will see if I can remove some of these now that it's working